lints/lint_ext_san_dns_name_too_long.go

```diff
@@ -0,0 +1,38 @@
+// lint_ext_san_dns_name_too_long.go
+
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type SANDNSTooLong struct {
+	// Internal data here
+}
+
+func (l *SANDNSTooLong) Initialize() error {
+	return nil
+}
+
+func (l *SANDNSTooLong) CheckApplies(c *x509.Certificate) bool {
+	return util.IsExtInCert(c, util.SANOID)
```

Comment:
from BR: 
This	extension MUST contain at least one entry.	 Each entry MUST be either a dNSName	containing the Fully‐Qualified Domain Name or an iPAddress containing the IP address	of a	server